### DISCUSSION TITLE
[Feature] Optional Image Version in docker-compose.yml

### DISCUSSION THREAD

**Discussion by twomoses** [immich-app/immich#2613] [Feature Request]
### The feature

I was looking for an easy way to suggest this without forking the entire repository, but I didn't find one.

It would be nice to include the option to pull a specific version or default to the release tag like so: 
```
  immich-web:
    container_name: immich_web
    ghcr.io/immich-app/immich-web:${IMMICH_VERSION:-release}
```
Then, if one would like to "pin" their version or ensure all containers are on the same version, they would add to the environment file: 
`IMMICH_VERSION=v1.59.0`

Those of us who try to update within just a few minutes of the release notification tend to get some updated images and some not updated, and it makes the app act very funny, especially when there are breaking changes from one version to the next. 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
I'd be fine with this personally. If you wanted to open a PR with the changes that would be great.

**Reply by twomoses** [reply]
Done! https://github.com/immich-app/immich/pull/2617

**Comment by jrasm91** [comment]
Implemented in #2617

Thanks for the contribution!
